src/align.cpp

```diff
@@ -518,7 +517,7 @@ void align_right_comments(void)
                        pc->orig_col, prev->orig_col_end, cpd.settings[UO_align_right_cmt_gap].n);
                skip = true;
             }
-            if (!skip)
+            if (skip == false)
```

Comment:
I think there are some arguments against it. First of all, the first one is faster to write. 

Secondly, we all know that if expressions are inherently verifying the truth of expressions so then why you wouldn't go as far as saying `if (skip == true)` ?

Also, the naming of the bool is a bit wrong as it should represent something positive, like `can_change` for instance.
So then, instead of having something like _if skip is false then_ you'll have `if can change then`.

It's nothing wrong with the code. It'll compile but it's a bit surprising. I've not seen this very often in production code. You don't have to update the PR just for this.